### PR TITLE
Standardizing Plant UML Diagrams

### LINKED ISSUE #573: GenPuml Diagrams shouldn't generate with spaces in file names
GenPuml Diagrams shouldn't generate with spaces in file names.

## References:
- TODO - Gen-PUML filename function

## AC:

- [x] The puml component diagrams have underscores instead of spaces in generated file names.
- [x] The puml object diagrams have underscores instead of spaces in generated file names.
- [x] The puml sequence diagrams have underscores instead of spaces in generated file names.

### PR DESCRIPTION
**Pull request by lizzcondrey**
# Description
Please include a summary of the changes and the related issue. List any dependencies that are required for this change.
- Standardized the file name generation for the three types of Plant UML diagrams

# Linked Items:
Please include the items resolved by and/or associated with this Pull Request using the following syntax to link them for closure upon completion.
Fixes #573 

## Type of change
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Feature improvement

### Feature Additions
Include pertinent changes as a result of adding the new feature.
- None

### Bug Fixes
Include pertinent changes of fixing the bug.
- Standardizing file name generation for Plant UML diagrams

### Breaking Changes
Include pertinent changes that qualify as breaking changes. Be sure to include information about the impact to end users and additional information for addressing the break if more than a version transition is necessary for mitigating.
- None

### Feature Improvement Changes
Include pertinent changes that are improvements to existing features.
- None

# Checklist:
- [x] Updated documentation accordingly.
- [ ] My changes generate no new warnings.
- [x] Have updated new and existing unit tests accordingly.
- [x] Linked associated items to be closed.
- [x] Have you bumped the version?

### CODE REVIEW

**Review by Coffee2Bits — APPROVED** [review]
Looks good! I just had to bump the dependency ["sphinxcontrib-serializinghtml == 1.1.5"](https://github.com/DevOps-MBSE/AaC/blob/1eac07958f089ebf0838aa0e719f03230497b1d0/python/setup.py#L56) to 1.1.9 to get the tests working in this branch and mine as well.

**Comment by lizzcondrey** [comment]
> Looks good! I just had to bump the dependency ["sphinxcontrib-serializinghtml == 1.1.5"](https://github.com/DevOps-MBSE/AaC/blob/1eac07958f089ebf0838aa0e719f03230497b1d0/python/setup.py#L56) to 1.1.9 to get the tests working in this branch and mine as well.

....This is what I get for finally posting this late after I had done all the work fighting with GitPod to get it back running for me. Because that was the fix for that portion also. Though I mitigated by swapping to `=> 1.1.5`. Forgot to bump that for the local branch since I can't push from GitPod right now. -.- Grrr.

**Comment by Coffee2Bits** [comment]
Looks like you just need to update a test to align with your fix. Nothing immediately stands out to me on the test, let me know if you'd like a hand tackling it.

```
======================================================================
FAIL: test_puml_object_diagram_to_file (tests.plugins.first_party.test_gen_plant_uml.TestGenPlantUml)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/AaC/AaC/python/tests/plugins/first_party/test_gen_plant_uml.py", line 106, in test_puml_object_diagram_to_file
    self.assertIn(os.path.basename(expected_puml_file_path), temp_directory_files)
AssertionError: 'tmpipj9ylvc_tmpipj9ylvc.puml' not found in ['tmpipj9ylvc_servicetwo.puml']
```

File: python/src/aac/__init__.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 import logging
 import os
 
-__version__ = "0.2.10"
+__version__ = "0.2.15"
```
Comment (cvchaparro) [inline python/src/aac/__init__.py:15]: We should make sure this is up-to-date before merging.

**Pull request merged**